backend/service/aws/aws.go

```diff
@@ -119,13 +131,26 @@ type client struct {
 type regionalClient struct {
 	region string
 
+	dynamodbCfg dynamodbConfig
+
 	s3          s3Client
 	kinesis     kinesisClient
 	ec2         ec2Client
 	autoscaling autoscalingClient
 	dynamodb    dynamodbClient
 }
 
+type dynamodbConfig struct {
+	scalingLimits scalingLimits
+}
+
+type scalingLimits struct {
+	maxReadCapacityUnits  int64
+	maxWriteCapacityUnits int64
+	maxScaleFactor        float32
+	enableOverride        bool
+}
```

Comment:
Will want to use our protos as the source of truth for this data structure, lets not redefined them in code here.